.github/workflows/e2e_test_for_releases.yml

```diff
@@ -2,19 +2,45 @@ name: AWS E2E for releases and pre-releases
 
 on:
   workflow_dispatch:
-  push:
-    tags:
-      - v*
-    paths-ignore:
-      - '**.md'
-      - 'cloudformation/*.yml'
-
-permissions:
-  id-token: write # This is required for requesting the JWT
-  contents: read  # This is required for actions/checkout
+  pull_request:
+    branches:
+      - main
+#  push:
+#    tags:
+#      - v*
 
 jobs:
+  check-changed-files:
```

Comment:
I don't sure that we need to skip these tests.
If we made a prerelease we should check the artifacts anyway in case if in github actions env changed something.

Reply:
Updated